android/src/main/java/com/reactnativecommunity/blurview/BlurViewManagerImpl.java

```diff
@@ -22,24 +24,18 @@ class BlurViewManagerImpl {
   public static final int defaultSampling = 10;
 
   public static @Nonnull BlurView createViewInstance(@Nonnull ThemedReactContext ctx) {
-    BlurView blurView = new BlurView(ctx);
+    BlurView blurView = new CustomBlurView(ctx);
     View decorView = Objects
       .requireNonNull(ctx.getCurrentActivity())
       .getWindow()
       .getDecorView();
     ViewGroup rootView = decorView.findViewById(android.R.id.content);
-    Drawable windowBackground = decorView.getBackground();
-    if (Build.VERSION.SDK_INT >= 31) {
-      blurView
-        .setupWith(rootView, new RenderEffectBlur())
```

Comment:
You will need to add this back for some things (like the bottom tabs). Basically, if the new prop with native id is set, use that view, otherwise use the root view